py3status/modules/xrandr.py

```diff
@@ -141,17 +156,28 @@ def _set_available_combinations(self):
         available_combinations = set()
         combinations_map = {}
 
+        if self.output_combinations is None:
+            whitelist = None
+        else:
+            whitelist = self.output_combinations.split('|')
+
         self.max_width = 0
         for output in range(len(self.layout['connected']) + 1):
             for comb in combinations(self.layout['connected'], output):
-                if comb:
-                    for mode in ['clone', 'extend']:
-                        string = self._get_string_and_set_width(comb, mode)
-                        if len(comb) == 1:
-                            combinations_map[string] = (comb, None)
-                        else:
-                            combinations_map[string] = (comb, mode)
-                        available_combinations.add(string)
+                if not comb:
```

Comment:
Ha, nice! We do come inside, but apparently only because `range` call starts with zero a couple of lines above, fixed :stuck_out_tongue_winking_eye: